Tests/Invoke-Tests.ps1

```diff
@@ -0,0 +1,45 @@
+<#
+    .SYNOPSIS
+    Runner to invoke Acceptance, Quality and / or Unit tests.
+
+    .DESCRIPTION
+    Test wrapper that invokes Acceptance, Quality and / or Unit tests.
+
+    .PARAMETER TestType
+    [Optional] The type of test that will be executed. The parameter value can be either All (default), Acceptance, Quality or Unit
+
+    .EXAMPLE
+    Invoke-Tests.ps1
+
+    .EXAMPLE
+    Invoke-Tests.ps1 -TestType Unit
+
+#>
+
+#Requires -Modules Pester
+#Requires -Modules PSScriptAnalyzer
```

Comment:
This can be formatted onto one line `#Requires -Modules Pester, PSScriptAnalyzer`

Reply:
Done 🥇